build/configs/artik053/hello/defconfig

```diff
@@ -0,0 +1,651 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
Hello includes only features which need to run hello App.
It's binary size is just 15KB and this can be a reference of minimal code footprint of TizenRT.